hack/vendor.sh

```diff
@@ -55,7 +55,7 @@ clone hg code.google.com/p/go.net 84a4013f96e0
 clone hg code.google.com/p/gosqlite 74691fb6f837
 
 #get libnetwork packages
-clone git github.com/docker/libnetwork 005bc475ee49a36ef2ad9c112d1b5ccdaba277d4
+clone git github.com/docker/libnetwork f72ad20491e8c46d9664da3f32a0eddb301e7c8d
```

Comment:
@duglin Yeah, I should have added a test case. Just that this change needs to go in 1.7 rc2, I felt some signs of urgency. If docker committer feel we can wait, I can add it.